teps/0042-taskrun-breakpoint-on-failure.md

```diff
@@ -134,14 +162,98 @@ failed step 0 as a success. Running this script would create `/tekton/tools/0.er
 `/tekton/debug/scripts/debug-breakpointexit` : Mark the step as completed with failure by writing to `/tekton/tools`. eg: User wants to exit
 breakpoint for failed step 0. Running this script would create `/tekton/tools/0` and `/tekton/tools/0.breakpointexit`.
 
-### User Stories
+## PipelineRun Proposal
```

Comment:
We might have to take in account pipeline timeouts as well, to avoid having the pipeline run controller changing the status of the taskrun while we debug.

Reply:
I will take these into consideration while writing the new TEP :eyeglasses: